dali/operators/reader/video_reader_decoder_cpu_op.cc

```diff
@@ -21,6 +21,7 @@ namespace dali {
 VideoReaderDecoderCpu::VideoReaderDecoderCpu(const OpSpec &spec)
     : DataReader<CPUBackend, VideoSampleCpu>(spec),
       has_labels_(spec.HasArgument("labels")) {
+      // TODO(awolant): Add stream support for loader
```

Comment:
I'm not sure if we can do that here. Stream is an executor property. `OpSpec` doesn't have it.

Reply:
This is stale comment just to indicate that I wanted to add stream passing to decoder. It is done via other means.
Removed the comment.

Reply:
Done, I hope :)